apps/nextjs-example/app/stylex.css

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+/**
+ * The @stylex directive is used by the @stylexjs/postcss-plugin.
+ * It is automatically replaced with generated CSS during builds.
+ */
+
+@layer stylex {
```

Comment:
This diverges from the README from `@stylexjs/postcss-plugin` because the reset uses CSS layers so I need to apply layers here as well.

I can think of two options:
a. Leave it as is and maybe add some explanation here
b Remove using CSS Layers entirely to align it with README.md

Reply:
Indeed, it was redundant and I tested before adding useCSSLayers option.
Fixed in https://github.com/facebook/stylex/pull/781/commits/e2ab3e3530c2e33250119e2e9b6347bed0d1e7e4